src/components/propositionalLogic/examplePage.tsx

```diff
@@ -2,75 +2,72 @@ import React from "react";
 
 import styles from "./propositionalLogic.module.scss";
 
+const exampleProofList: Array<Array<[string, string, boolean]>> = [
+	[
+		["Wir zeigen: a -> (b -> c) -> (a und b) -> c.", "Dies soll bewiesen werden.", false],
+		["Es seien a, b und c Aussagen.", "Allgemeine Vorraussetzung", false],
+		["Angenommen $a ->(b -> c)$.", "Vorraussetzung: Die Prämisse a -> (b -> c) ist wahr.", false],
+		["Angenommen ferner es gilt $(a und b)$.", "Vorraussetzung: (a und b) ist wahr.", false],
+		["Dann folgt a.", "a folgt, da (a und b) nur dann wahr ist, wenn a wahr ist.", false],
+		["Ausserdem folgt b.", "b folgt, da (a und b) nur dann wahr ist, wenn b wahr ist.", false],
+		["Damit gilt $(b -> c)$.", "Die Konlusion (b -> c) gilt, da die Prämissen a,sowie (a und b) beide gelten.", false],
+		["Ferner folgt c.", "Die Konlusion c gilt, da die Prämissen a, sowie (a und b) beide gelten.", false],
+		["Also gilt $a -> (b -> c) -> (a und b) -> c$.", "qed", false]],
+	[
+		["test1", "test2", false],
+		["test3", "test4", false]],
+];
+
 interface State {
-	proofPart: Array<[string, string, boolean]>;
+	exampleProofList: Array<Array<[string, string, boolean]>>;
 }
 
 class ExamplesPropositionalLogic extends React.Component<{}, State> {
 
 	// Via constructor since an immediate assignment led to type issues
 	constructor(props: {}) {
 		super(props);
-		const proofPart: Array<[string, string, boolean]> = [
-			["Es seien a, b und c Aussagen.", "Allgemeine Vorraussetzung", false],
-			["Angenommen $a ->(b -> c)$.", "Vorraussetzung: Die Prämisse a -> (b -> c) ist wahr.", false],
-			["Angenommen ferner es gilt $(a und b)$.", "Vorraussetzung: (a und b) ist wahr.", false],
-			["Dann folgt a.", "a folgt, da (a und b) nur dann wahr ist, wenn a wahr ist.", false],
-			["Ausserdem folgt b.", "b folgt, da (a und b) nur dann wahr ist, wenn b wahr ist.", false],
-			["Damit gilt $(b -> c)$.", "Die Konlusion (b -> c) gilt, da die Prämissen a,sowie (a und b) beide gelten.", false],
-			["Ferner folgt c.", "Die Konlusion c gilt, da die Prämissen a, sowie (a und b) beide gelten.", false],
-		];
 
-		this.state = { proofPart };
+		this.state = { exampleProofList };
 	}
 
 	public render() {
 		return <div className={styles.site}>
-			<table>
+			<button>Nächstes Beispiel</button>
```

Comment:
This would need an onClick event so it might look like this:
`<button onClick={showNextExample()}`
And the function would need to reset the "clicked" array and increase the "exampleToShow" variable by one. These changes have to be put into the `this.setState({...})` methode. Generally you only wanna call it when you change a variable in the state which should change the look of the website.